pom.xml

```
@@ -4,19 +4,20 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.30</version>
+        <version>4.46</version>
+        <relativePath />
     </parent>
 
     <artifactId>artifactdeployer</artifactId>
     <packaging>hpi</packaging>
-    <name>Jenkins Artifact Deployer Plug-in</name>
+    <name>Jenkins Artifact Deployer Plugin</name>
     <version>1.3-SNAPSHOT</version>
-    <url>http://wiki.jenkins-ci.org/display/JENKINS/ArtifactDeployer+Plugin</url>
+    <url>https://plugins.jenkins.io/artifactdeployer/</url>
```

This line either needs to be changed back to its previous value or a new file needs to be created.

The plugin documentation is currently being read from the [wiki archive](https://raw.githubusercontent.com/jenkins-infra/plugins-wiki-docs/master/artifactdeployer/README.md) by the plugins site.  If README.md is created in this pull request with the content from the wiki archive, then this line should be:
```suggestion
    <url>https://github.com/jenkinsci/artifactdeployer/</url>
```